packages/devtools_app/lib/src/vm_developer/vm_statistics_view.dart

```diff
@@ -0,0 +1,295 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:vm_service/vm_service.dart';
+
+import '../common_widgets.dart';
+import '../table.dart';
+import '../table_data.dart';
+import '../utils.dart';
+import 'vm_developer_common_widgets.dart';
+import 'vm_developer_tools_screen.dart';
+import 'vm_service_private_extensions.dart';
+import 'vm_statistics_view_controller.dart';
+
+class VMStatisticsView extends VMDeveloperView {
+  const VMStatisticsView()
+      : super(
+          id,
+          title: 'VM',
+          icon: Icons.devices_sharp,
+        );
+  static const id = 'vm-statistics';
+
+  @override
+  Widget build(BuildContext context) => VMStatisticsViewBody();
+}
+
+/// Displays general information about the state of the Dart VM.
+class VMStatisticsViewBody extends StatelessWidget {
+  final controller = VMStatisticsViewController();
+  @override
+  Widget build(BuildContext context) {
+    return ValueListenableBuilder(
+      valueListenable: controller.refreshing,
+      builder: (context, refreshing, _) {
+        return Column(
+          children: [
+            Row(
+              children: [
+                RefreshButton(
```

Comment:
does this need to be in a row? if you take it out of the row, you can tweak the column alignment to put the refresh button where you want it

Reply:
The problem I run into when removing the row is that the button decides it wants to take up the full width of the page. Not sure if there's another way around that?

Reply:
Ah interesting. Can't think of a a better solution off the top of my head.